apps/web/src/views/Home/components/Banners/hooks/useMultipleBannerConfig.tsx

```diff
@@ -40,6 +41,10 @@ export const useMultipleBannerConfig = () => {
 
   return useMemo(() => {
     const NO_SHUFFLE_BANNERS: IBannerConfig[] = [
+      {
+        shouldRender: true,
+        banner: <PCSXBanner />,
+      },
```

Comment:
Move this under `banner: <UserBanner />`

Reply:
done